website/content/en/docs/faqs/_index.md

```diff
@@ -172,3 +172,38 @@ For more information and examples, please see the type-specific docs:
 [rbac]:https://kubernetes.io/docs/reference/access-authn-authz/rbac/
 [scorecard-doc]: https://sdk.operatorframework.io/docs/testing-operators/scorecard/
 [project-doc]: /docs/overview/project-layout
+
+## Preserve the `preserveUnknownFields` in your CRDs
+
+The `preserveUnknownFields` will be removed if set to false when running `make bundle`. Because of some underlying data structure changes and how yaml is unmarshalled, it is best to add them back in after they have been written.
```

Comment:
let's add link to when preserveUnknownFields needs to be set to false explicitly
```suggestion
The [`preserveUnknownFields`](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#field-pruning) will be removed if set to false when running `make bundle`. Because of some underlying data structure changes and how yaml is unmarshalled, it is best to add them back in after they have been written.
```

Reply:
Modified according to comments.